roles/openshift_repos/tasks/main.yaml

```diff
@@ -33,14 +34,16 @@
     state: absent
   with_fileglob:
   - '*/repos/*'
-  when: not (item | search("/files/" ~ openshift_deployment_type ~ "/repos"))
+  when: not (item | search("/files/" ~ openshift_deployment_type ~ "/repos")) and not is_containerized | bool
 
 - name: Configure gpg keys if needed
   copy: src={{ item }} dest=/etc/pki/rpm-gpg/
   with_fileglob:
   - "{{ openshift_deployment_type }}/gpg_keys/*"
+  when: not is_containerized | bool
 
 - name: Configure yum repositories
   copy: src={{ item }} dest=/etc/yum.repos.d/
   with_fileglob:
   - "{{ openshift_deployment_type }}/repos/*"
+  when: not is_containerized | bool
```

Comment:
Maybe we should just break the dependency on openshift_repos all together and make it a conditional include from the playbooks.